2015-09-31-habitica-habit-building-program.md

```diff
@@ -0,0 +1,204 @@
+up on them (missing features, not intuitive enough, or simply because I couldn't
+build the habit of checking them regularly and forgot them). But when I started
+using Habitica back in its beta days, although it was cribbled with bugs, I
+sticked to it because I was having fun and had met very friendly people there.
+
+{: .clear}
+![Habitica Logo](/images/habitica/logo.png "New logo looking like a Gryphon"){: .float_left}
+Formely named **HabitRPG**, the application was renamed **Habitica** on July 31th 2015.
+The old name was difficult to pronunce and confused by people
+(HabitRG? HabitGRG?), so they decided to name the application after the land of
+Habitica, where all these adventures take place.
+
+The [Gamification](https://en.wikipedia.org/wiki/Gamification) comes in
+multiples aspects:
+
+* **Health**, **Experience**, **Mana**: Like any [RPG
+  game](https://en.wikipedia.org/wiki/Role-playing_game), your avatar will
+  evolve as you complete tasks, and will die if you're slacking off.
+
+  {: .centered}
+  ![Health, XP and Mana bars](/images/habitica/health_xp_mana.png)
+* **Attribute points and Class System**: Do you want to complete quests faster,
+  get more gold, avoid dying? Smartly distribute your points in [Strenght,
+  Intelligence, Constitution and Perception](http://habitica.wikia.com/wiki/Character_Attributes),
+  and select the most appropriate class for your avatar
+  ([Warrior](http://habitica.wikia.com/wiki/Character_Attributes),
+  [Mage](http://habitica.wikia.com/wiki/Mage),
+  [Rogue](http://habitica.wikia.com/wiki/Rogue) or 
+  [Healer](http://habitica.wikia.com/wiki/Healer))
+
+  {: .centered }
+  ![Classes system: Warrior, Mage, Rogue and Healer](/images/habitica/classes.png)
+* **Gold**, **Food**, **Eggs**, **Potions**: As you complete your tasks, items
+  will drop.
+ 
+  {: .centered }
+  ![items](/images/habitica/items.png)
+* **Pets**, **Mounts**, **Equipment**, **Background**: Hatch eggs into pets,
+  feed pets into mounts, buy equipments, and enjoy customizing your avatar in
+  any way you'd like.
+* **Party**, **Guilds**: You can join groups of people from all over the world,
+  doing completely different jobs, and socialize with them, motivate each other
+  to become everyday a bit more productive.
+
+  {: .centered}
+  ![Party backgrounds](/images/habitica/party.png)
+* **Quests**, **Challenges**: Fight against bosses by completing your tasks,
+  together with your team members, or compete against other players for habit
+  building activities.
+* **Achievements badges**: You can collect badges by contributing, by killing
+  bosses, by getting all pets/mounts/equipments, by interacting with your party,
+  and more!
+
+  {: .centered}
+  ![Achievement badges](/images/habitica/badges.png)
+
+
+# An excerpt of my Personal experience
+
+What hooked me up the most are the facts that Habitica is Open Source and
+strongly motivates you to contribute, as well as the community.
```

Comment:
What hooked me up the most was the community and that Habitica is open source and strongly encourages you to contribute.